src/xunitSettings.props

```diff
@@ -0,0 +1,8 @@
+<Project>
+  <!-- use the xunit JSON settings file -->
+  <ItemGroup>
```

Comment:
This can be used in other test suites too - the Akka.Remote ones have the most flaky behavior aside from a handful of Akka.Streams specifications.